### PR TITLE
Search

### PR DESCRIPTION
**Pull request by leftclickben**
RWAHS-308

### CODE REVIEW

**Comment by leftclickben** [comment]
There's quite a bit here, happy to go through it with you.

**Comment by leftclickben** [comment]
One thing I forgot about this is that your web server running providence needs to allow CORS from the relevant domain, in dev, localhost:8888. There are 2 directives for Apache, one to enable and I've to set the allowed origin header.  You also need to enable headers module in Apache.

File: public/app/ui/pages/search/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                            <div class="btn-group btn-group-justified">
+                                <!-- ko foreach: searchTypes -->
+                                <a class="btn btn-default" data-bind="html: label, css: { active: active }, click: makeActive"></a>
+                                <!-- /ko -->
+                            </div>
+                        </div>
+                    </div>
+                    <div class="panel-footer text-right">
+                        <div class="btn-group">
+                            <button type="reset" class="btn btn-default">
```
Comment (kehh) [inline public/app/ui/pages/search/index.html]: This doesn't reset the `searchTypes` buttons

**Comment by leftclickben** [comment]
This now uses the modified format of `Creator` as given by rwahs/providence-configuration!21.

**Pull request merged**